bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx

```diff
@@ -193,7 +210,17 @@ class ActionsDropdown extends Component {
             label={formatMessage(createBreakoutRoom)}
             description={formatMessage(createBreakoutRoomDesc)}
             key={this.createBreakoutRoomId}
-            onClick={this.handleCreateBreakoutRoomClick}
+            onClick={this.onCreateBreakouts}
+          />
+        )
+        : null),
+      (isUserModerator && !meetingIsBreakout && hasBreakoutRoom && getUsersNotAssigned(users).length
```

Comment:
assign this a variable with a meaningful name `canInviteUsers` or something like that